src/components/BookList.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BookList = ({ books, onDelete }) => (
+  <div>
+    {books.map((book) => (
+      <div key={book.id}>
+        <h3>
+          Title:
+          {book.title}
+        </h3>
+        <p>
+          Author:
+          {book.author}
+        </p>
+        <button type="submit" onClick={() => onDelete(book.title)}>Delete</button>
+      </div>
+    ))}
+  </div>
```

Comment:
- [ ] I'd suggest rendering each book with a separate `<Book />` component that takes the book data as a prop and renders the title and the author along with the button for removing each book.
- This was part of the project requirements shown below:
  ![image](https://github.com/nilab-khwaja/bookstore/assets/104968236/f6d8e6a7-2aa4-4dac-8ed2-37bb41c5f959)